examples/using-multiple-themes/src/gatsby-plugin-theme-ui/index.js

```diff
@@ -0,0 +1,17 @@
+import merge from "deepmerge"
```

Comment:
Maybe add a comment that `deepmerge` is available through gatsby-theme-blog?

Reply:
This is going to be updated shortly after a bug fix for theme-ui's "merge", so I think it's ok for now.